test/e2e/framework/network/utils.go

```diff
@@ -53,8 +53,9 @@ const (
 	// EndpointUDPPort is an endpoint UDP port for testing.
 	EndpointUDPPort = 8081
 	// EndpointSCTPPort is an endpoint SCTP port for testing.
-	EndpointSCTPPort      = 8082
-	testContainerHTTPPort = 8080
+	EndpointSCTPPort = 8082
+	// testContainerHTTPPort is the test container http port.
+	testContainerHTTPPort = 9080
```

Comment:
this changed from 8080 to 9080, expected?

Reply:
this container is wrongly using the same ports that the other containers that are endpoints from the service under test.
It is a different container that expose a handler to execute commands, not to be polled

Reply:
IIRC I added it in a separate commit and the description should add a bit more of detail